CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-10270: The `RenamePolicyCategory` and `DeletePolicyCategory` methods in the
 `v1/policycategories` endpoint have been deprecated, and will be removed in future releases. 
   - For questions about this change, please contact the Red Hat support team at support@redhat.com.
+- ROX-10018: The policy `OpenShift: Kubeadmin Secret Accessed` will not trigger for the default OpenShift `oauth-apiserver-sa` service account as that is an expected access pattern for the apiserver on OpenShift.
```

Comment:
It reads like it won't whether or not an exclusion exists.

Reply:
Updated wording